src/PanZoom.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react'
-
+import warning from 'warning';
```

Comment:
I haven't set any linting rules, but as the codebase doesn't have any semi-colon, I think it would be good to follow this syntax.

Reply:
```suggestion
import warning from 'warning'
```

Reply:
@mnogueron If you want I can create a new pull request for prettier and eslint after this one 😊

Reply:
@n1ru4l If you don't mind I would like to handle these modifications :) I've never worked with Prettier and I think it would be good for me to read about it